### ISSUE TITLE
Downloading a specific KaRIn L2_LR_SSH product

### ISSUE THREAD

**Issue by shoshanareich** [podaac/data-subscriber#151]
Is there an argument in the podaac_data_downloader to download just a specific L2_LR_SSH product, i.e. basic, expert, wind wave, or unsmoothed?

**Comment by selipot** [comment]
Indeed, I would like to see this as well. I would like to download only the Basic files.

**Comment by reint-fischer** [comment]
I agree that this would be very useful. Additionally, I was wondering if it would be possible to add an option to select individual passes. We can already specify the cycle, but I am interested in specific passes only.
